SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+## Supported Versions
+
+See releases section for versions of Alby that are currently being supported with 
+security updates: https://github.com/getAlby/lightning-browser-extension/tags
```

Comment:
```suggestion
security updates: https://github.com/getAlby/lightning-browser-extension/releases
```

Reply:
oops! thanks for pointing this out. Resolved!